Code/Framework/AzFramework/Platform/Windows/AzFramework/Process/ProcessWatcher_Win.cpp

```diff
@@ -14,6 +14,7 @@
 #include <AzFramework/Process/ProcessWatcher.h>
 #include <AzFramework/Process/ProcessCommunicator.h>
 #include <AzFramework/Process/ProcessCommon.h>
+#include <AzFramework/StringFunc/StringFunc.h>
```

Comment:
Can you change this include to `<AzCore/StringFunc/StringFunc.h>`?

```suggestion
#include <AzCore/StringFunc/StringFunc.h>
```